workspaces/announcements/plugins/announcements/src/components/Admin/AnnouncementsContent/AnnouncementsContent.tsx

```diff
@@ -199,21 +200,43 @@ export const AnnouncementsContent = () => {
     },
     {
       title: (
-        <Typography>{t('admin.announecementsContent.table.status')}</Typography>
+        <Typography>{t('admin.announcementsContent.table.status')}</Typography>
       ),
       sorting: true,
       field: 'category',
       render: rowData =>
         rowData.active
-          ? t('admin.announecementsContent.table.active')
-          : t('admin.announecementsContent.table.inactive'),
+          ? t('admin.announcementsContent.table.active')
+          : t('admin.announcementsContent.table.inactive'),
     },
     {
       title: (
         <Typography>
-          {t('admin.announecementsContent.table.actions')}
+          {t('admin.announcementsContent.table.created_at')}
         </Typography>
       ),
+      sorting: true,
```

Comment:
`sorting: true` may be the default if you want to leave that off your additions. An additional PR would be needed to clean up the rest